common/app/model/content/Atoms.scala

```diff
@@ -374,6 +414,39 @@ object Quiz extends common.Logging {
   }
 }
 
+object StoryQuiz {
+  def extractImage(quiz: atomapi.quiz.QuizAtom): Option[String] = for {
+    firstQuestion <- quiz.content.questions.headOption
+    firstAsset <- firstQuestion.assets.headOption
+    json = Json.parse(firstAsset.data)
+    assets = json \ "assets"
+    widths = (assets \\ "width").map(_.as[Int])
+    width140 = widths.indexWhere(_ == 140)
+    url <- (assets(width140) \ "secureUrl").asOpt[String]
```

Comment:
Yeah I thought about that. This is a test and no plan to make this a permanent feature, so I can live with that for now 👍